skuba-update/.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
+# line length is intentionally set to 80 here because:
+# 1) we can use Bugbear.  See for example what black is doing:
+# https://github.com/psf/black/blob/master/README.md#line-length
+# 2) That's very pedantic.
+max-line-length = 80
```

Comment:
Personally, I think 80 is too short.  But looking at the .py code I see that skuba_update is already at 80 characters (with the exception of the long description in setup.py which is removed in this PR) so this should be fine.

Reply:
Yes, I think 88 is the recommended for black, but I didn't want to raise this yet :)